ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestionMapping.java

```diff
@@ -33,16 +34,38 @@ public IngestionMapping(String ingestionMappingReference, IngestionMappingKind i
     }
 
     /**
-     * Sets the ingestion mapping parameters
+     * Please use setIngestionMappingReference for production as passing the mapping every time is wasteful
+     * @param ingestionMapping          Array of columnMappings of the same kind.
+     * @param ingestionMappingKind      IngestionMappingKind: the format of the source data to map from.
+     */
+    public IngestionMapping(ColumnMapping[] ingestionMapping, IngestionMappingKind ingestionMappingKind) {
```

Comment:
naming here is strange. parameter name should probably be columnsMapping instead